app/views/pets/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>All Pets</h2>
+  <%@pets.each do |pet|%>
+  <img src="<%=  pet.image %>" alt="<%= pet.image %>" style="width:200px">
```

Comment:
tried out `image-path`; had a lot of errors
https://www.w3schools.com/html/html_filepaths.asp
- for different versions